eland/cli/eland_import_hub_model.py

```diff
@@ -238,13 +239,13 @@ def check_cluster_version(es_client, logger):
         )
         exit(1)
 
-    # PyTorch was upgraded to version 2.1.2 in 8.13
+    # PyTorch was upgraded to version 2.3.1 in 8.15.1
     # and is incompatible with earlier versions
-    if major_version == 8 and minor_version < 13:
+    if major_version == 8 and minor_version < 15 and patch_version < 1:
```

Comment:
If the user is on 8.15.0, this will translate to: `if True and False and True` and they won't get the error. I would suggest using `sem_ver` directly instead:

```suggestion
    if sem_ver < (8, 15, 1):
```

Reply:
Thanks. I've made this change and updated the version check to 8.15.2